src/gui/transferlistfilterswidget.cpp

```diff
@@ -66,6 +68,27 @@ namespace
             scheme = "http";
         return scheme;
     }
+
+    class ArrowCheckBox : public QCheckBox
+    {
+    public:
+        using QCheckBox::QCheckBox;
+        void paintEvent(QPaintEvent *event) override
+        {
+            QPainter painter(this);
+
+            QStyleOption indicatorOption;
+            indicatorOption.initFrom(this);
+            indicatorOption.rect = style()->subElementRect(QStyle::SE_CheckBoxIndicator, &indicatorOption, this);
+            style()->drawPrimitive(isChecked() ? QStyle::PE_IndicatorArrowDown : QStyle::PE_IndicatorArrowUp,
```

Comment:
```c++
            style()->drawPrimitive((isChecked() ? QStyle::PE_IndicatorArrowDown
                                                : (QGuiApplication::isLeftToRight() ? QStyle::PE_IndicatorArrowRight : QStyle::PE_IndicatorArrowLeft))
                                   , &indicatorOption, &painter, this);
```
Fixes wrong formatting and personally I like PE_IndicatorArrowRight better :P

Reply:
> I like PE_IndicatorArrowRight better
will change

running uncrustify changes some other parts of the files as well(not part of this PR), can i run it

PS: you guys should shift